aerleon/aclcheck_cmdline.py

```diff
@@ -65,9 +69,15 @@ def main():
         help='Protocol (tcp, udp, icmp, etc.)',
     )
     _parser.add_argument(
-        '--dport', '--destination-port', dest='destination_port', help='Destination port.'
+        '--dport',
+        '--destination-port',
+        '--destination_port',
+        dest='destination_port',
+        help='Destination port.',
+    )
+    _parser.add_argument(
```

Comment:
Did this not get formatted automatically? Can it be formatted like above?

Reply:
This is how the auto formatter (black) chose to format the code.